src/test/java/com/verygood/security/coding/AliasesServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package com.verygood.security.coding;
+
+import com.verygood.security.coding.aliases.AliasesService;
+import com.verygood.security.coding.aliases.AliasesServiceImpl;
+import com.verygood.security.coding.dao.MockAliasRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@RunWith(SpringRunner.class)
+@Slf4j
+public class AliasesServiceTest {
+    private AliasesService aliasesService = new AliasesServiceImpl(new MockAliasRepository());
+
+    @Test
+    public void inputIsEncryted() {
```

Comment:
Not sure we need `@RunWith(SpringRunner.class)` here.
We don't need Spring context to test AliasesService logic.